ledger/light/proc/set_code.go

```diff
@@ -104,10 +105,12 @@ func (p *SetCode) Proceed(ctx context.Context) error {
 	material := record.Material{
 		Virtual: &virtual,
 	}
+
 	err = p.dep.records.Set(ctx, p.recordID, material)
 	if err != nil {
 		return errors.Wrap(err, "failed to store record")
 	}
+	inslogger.FromContext(ctx).WithField("code_id", p.recordID.DebugString()).Infof("saved code")
```

Comment:
this is wrong use of WithField

Reply:
Why? I see a lot of same usage in project.

Reply:
This code is equal to `inslogger.Info("saved code ", p.recordID.DebugString())`, I find it wrong to use WithField for one statement.